compose/ui/ui-text/src/uikitMain/kotlin/androidx/compose/ui/text/input/PlatformImeOptions.skiko.uikit.kt

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.text.input
+
+import platform.UIKit.UIKeyboardAppearance
+import platform.UIKit.UIKeyboardAppearanceDefault
+import platform.UIKit.UIKeyboardType
+import platform.UIKit.UIKeyboardTypeDefault
+import platform.UIKit.UIReturnKeyType
+import platform.UIKit.UITextAutocapitalizationType
+import platform.UIKit.UITextAutocorrectionType
+import platform.UIKit.UITextContentType
+
+private class PlatformImeOptionsImpl(
+    val keyboardType: UIKeyboardType?,
+    val keyboardAppearance: UIKeyboardAppearance,
+    val returnKeyType: UIReturnKeyType?,
+    val textContentType: UITextContentType,
+    val isSecureTextEntry: Boolean?,
+    val enablesReturnKeyAutomatically: Boolean,
+    val autocapitalizationType: UITextAutocapitalizationType?,
+    val autocorrectionType: UITextAutocorrectionType?,
+    val hasExplicitTextContentType: Boolean
+): PlatformImeOptions()
+
+/**
+ * Configuration for creating instances of PlatformImeOptions.
+ */
+class PlatformImeOptionsConfiguration internal constructor() {
```

Comment:
Public API shape LGTM, but Iet's play safe here and mark all new public APIs as `@ExperimentalComposeUI` for one version. Just to let us change it without "breaking changes"